app/FreshRSS.php

```diff
@@ -32,6 +32,14 @@ public function init() {
 			die($message);
 		}
 
+		if (FreshRSS_Context::$system_conf->logo_html != '') {
+			// Relax Content Security Policy to allow external images if a custom logo HTML is used
+			Minz_ActionController::_defaultCsp([
```

Comment:
Instead of changing the default CSP, I would rather set only the `img-src` policy. The `_csp()` method could be adapted to allow to change a specific policy (something like `$this->csp_policies = array_merge($this->csp_policies, $policies);`), but you would need to remove the default CSP policy warning as well (which is fine to me).

Reply:
I agree that the CSP methods could be made better, but editing (e.g. adding lines / merging) will likely open a number of cases, which I would rather keep to another PR. For instance, it could be nice to allow something for one domain, and then allow the same for another domain in two different calls (e.g. for extensions).
Having a way to set the global default CSP is still nice in my opinion, to avoid having to set it for each controller.

Reply:
I'm always a bit reluctant to change default values (I feel like we should avoid that as much as possible), but it's good enough :+1: